careerfairsystems/arkad-app
# Loading save

## Change description

Now properly shows loading and successfully saved when a students info is being saved. Also changed color of loading indicator to arkad blue. 

## How to verify

Try to save new rating for student in company login. Loading indicator should show and when loading is finished show success symbol. 

## Issues fixed

#360
